js/script.js

```diff
@@ -85,10 +126,39 @@ class Store {
 
     localStorage.setItem('books', JSON.stringify(books));
   }
+
+  static UpdateBook(book) {
+    const books = Store.getBooks();
+
+    books.forEach((b, index) => {
```

Comment:
It is cleaner and more efficient to use[ `Array.prototype.find`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) here.